### ISSUE TITLE
Recently saved items not accessible via Voice Control

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#13112]
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-5690)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on v9000 (27761)
